cdap-examples/FileSetExample/src/test/java/co/cask/cdap/examples/fileset/FileSetWordCountTest.java

```diff
@@ -56,7 +56,8 @@ public void testWordCountOnFileSet() throws Exception {
     final String line2 = "b a b";
 
     // discover the file set service
-    ServiceManager serviceManager = applicationManager.startService("FileSetService");
+    ServiceManager serviceManager = applicationManager.getServiceManager("FileSetService");
+    serviceManager.start();
```

Comment:
You can have the `start()` method returns the manager so that you can chain calls.

```java
ServiceManager serviceManager = applicationManager.getServiceManager("FileSetService").start();
```